crates/storage-api/src/invocation_status_table/mod.rs

```diff
@@ -351,6 +351,25 @@ impl InvocationStatus {
 
 protobuf_storage_encode_decode!(InvocationStatus, crate::storage::v1::InvocationStatusV2);
 
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum InvocationStatusDiscriminants {
+    Scheduled,
+    Inboxed,
+    Invoked,
+    Suspended,
+    Killed,
+    Completed,
+}
+
+/// Lite status of an invocation.
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct InvocationLite {
+    pub status: InvocationStatusDiscriminants,
+    pub invocation_target: InvocationTarget,
+}
+
+protobuf_storage_encode_decode!(InvocationLite, crate::storage::v1::InvocationV2Lite);
```

Comment:
The only reason for this type to be here is the package organization, otherwise i could have hid everything in the partition-store module (and i could have even avoided defining the Lite type in the domain.proto).